baseboards/riscv-gvsoc.exp

```diff
@@ -0,0 +1,28 @@
+# Load the generic configuration for this board.  This will define any
+# routines needed to communicate with the board.
+load_generic_config "sim"
+
+set_board_info sim "gvsoc_sim"
+set_board_info sim,options ""
+set_board_info is_simulator 1
+
+# No default multilib options are needed for this board.
+process_multilib_options ""
+
+set_board_info cflags    ""
+
+set_board_info ldflags   "$::env(PULP_SDK_HOME)/libs/libbasic-runtime.a -nostartfiles -Wl,--gc-sections -L$::env(PULP_SDK_HOME)/rtos/pulpos/pulp/kernel -Tchips/pulp/link_huge_stack_model.ld -lgcc"
```

Comment:
That seems too PULP specific and highly depend on the folder layout of PULP SDK? so I am not sure it's make sense to merge that rather than just keep in PULP SDK and set `DEJAGNULIBS`.

Reply:
Thanks Kito Cheng, 

I have two questions: I don't undertand how it shoud work removing ldflags from riscv-gvsoc.exp and leaving them only in PULP_SDK, since ldflags are needed at compile time by DejaGNU.
Furthermore, we need to set `DEJAGNULIBS` environment variable appropriately if `expect` and `runtest` are not in the `PATH`. But our `expect` and `runtest` are already in the `PATH`.